hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/statemachine/DatanodeConfiguration.java

```diff
@@ -42,9 +42,9 @@ public class DatanodeConfiguration {
       "hdds.datanode.container.delete.threads.max";
   static final String PERIODIC_DISK_CHECK_INTERVAL_MINUTES_KEY =
       "hdds.datanode.periodic.disk.check.interval.minutes";
-  public static final String FAILED_DATA_VOLUMES_TOLERATED_KEY =
```

Comment:
I think the current name is fine, `PERIODIC_DISK_CHECK_INTERVAL_MINUTES_KEY` and `FAILED_DB_VOLUMES_TOLERATED_KEY` are also using this naming scheme.

You can tell it's datanode configuration from the class name `DatanodeConfiguration`.

Reply:
@DaveTeng0 I agree with @kaijchen, I think this change is unnecessary.

Reply:
Oh make sense!! Sorry I'll close this PR. thank you guys!